packages/salesforcedx-apex-debugger/src/context/isvContextUtil.ts

```diff
@@ -5,16 +5,14 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {
-  ForceConfigGet,
-  GlobalCliEnvironment
-} from '@salesforce/salesforcedx-utils-vscode';
 import {
   ENV_SFDX_DEFAULTUSERNAME,
   ENV_SFDX_INSTANCE_URL,
+  ForceConfigGet,
+  GlobalCliEnvironment,
   SFDX_CONFIG_ISV_DEBUGGER_SID,
   SFDX_CONFIG_ISV_DEBUGGER_URL
-} from '@salesforce/salesforcedx-utils-vscode';
+} from '@salesforce/salesforcedx-utils';
```

Comment:
The debugger should only depend on utils.  Not sure how this code through previously.